test/OpenTelemetry.Instrumentation.GrpcClient.Tests/OpenTelemetry.Instrumentation.GrpcClient.Tests.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>Unit test project for OpenTelemetry GrpcClient instrumentation</Description>
+    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Protobuf Include="Proto\greet.proto" GrpcServices="Client" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
+    <PackageReference Include="Moq" Version="4.11.0" />
+    <PackageReference Include="xunit" Version="2.4.1" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
+    </PackageReference>
+    <DotNetCliToolReference Include="dotnet-xunit" Version="2.3.1" />
+
+    <PackageReference Include="Google.Protobuf" Version="3.12.3" />
+    <PackageReference Include="Grpc.Core" Version="2.30.0" />
+    <PackageReference Include="Grpc.Net.Client" Version="2.30.0-pre1" />
+    <PackageReference Include="Grpc.Tools" Version="2.30.0" PrivateAssets="All" />
```

Comment:
Change the version to 2.25.0

I believe the build error you're getting in CI is because of different versions of Grpc.Tools. There are a bunch of ways to fix this problem, but the simplest right now is using the same version of Grpc.Tools everywhere in a solution.

Error:

> ##[error]/home/vsts/.nuget/packages/grpc.tools/2.30.0/build/_protobuf/Google.Protobuf.Tools.targets(64,5): Error MSB4062: The "Grpc.Tools.ProtoToolsPlatform" task could not be loaded from the assembly /home/vsts/.nuget/packages/grpc.tools/2.30.0/build/_protobuf/netstandard1.3/Protobuf.MSBuild.dll. Assembly with same name is already loaded Confirm that the <UsingTask> declaration is correct, that the assembly and all its dependencies are available, and that the task contains a public class that implements Microsoft.Build.Framework.ITask.


Other place Grpc.Tools is used:

https://github.com/open-telemetry/opentelemetry-dotnet/blob/9f3bb81b4a96c8b7f57705a0e4bef1b85b8c97bc/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OpenTelemetry.Exporter.OpenTelemetryProtocol.csproj#L12

Reply:
Thanks! Using 2.25.0 did the trick.